src/agents/onboarding_agent.py

```diff
@@ -26,12 +26,20 @@
 
 
 def _is_allowed_by_moderation(user_input: str, openai_api_key: str) -> bool:
```

Comment:
Change is to include:

- empty input
- open ai failure

as "permitted" responses